src/words/ast/INodeAssign.java

```diff
@@ -8,9 +10,36 @@ public INodeAssign(Object... children) {
 		super(children);
 	}
 
+	/**
+	 * Always returns null. Throws WordsRuntimeException for called methods, including WordsObjectNotFoundException.
+	 * Side effect of assigning a property to an object.
+	 */
 	@Override
 	public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException {
-		// TODO
-		throw new AssertionError("Not yet implemented");
+		WordsObject obj = children.get(0).eval(environment).objValue;
```

Comment:
We could put in an assert here that the obj is not null.  In principle. the child (a reference list) could have been NOTHING, but the grammar right now assures that that's not the case.  If we change the grammar, we might miss that.  I guess we'd catch that regardless via a null pointer exception, but an assertion with a comment might be clearer.